addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionOracleTimingScanRule.java

```diff
@@ -62,58 +68,125 @@
+        ORIG_VALUE_TOKEN
+                + ") and exists ("
+                + ORACLE_SLEEP_FUNCTION
+                + ")"
+                + ORACLE_ONE_LINE_COMMENT, // Param in WHERE clause somewhere
+        ORIG_VALUE_TOKEN
+                + " or exists ("
+                + ORACLE_SLEEP_FUNCTION
+                + ")"
+                + ORACLE_ONE_LINE_COMMENT, // Param in WHERE clause somewhere
+        ORIG_VALUE_TOKEN
+                + "' or exists ("
+                + ORACLE_SLEEP_FUNCTION
+                + ")"
+                + ORACLE_ONE_LINE_COMMENT, // Param in WHERE clause somewhere
+        ORIG_VALUE_TOKEN
+                + "\" or exists ("
+                + ORACLE_SLEEP_FUNCTION
+                + ")"
+                + ORACLE_ONE_LINE_COMMENT, // Param in WHERE clause somewhere
+        ORIG_VALUE_TOKEN
+                + ") or exists ("
+                + ORACLE_SLEEP_FUNCTION
+                + ")"
+                + ORACLE_ONE_LINE_COMMENT, // Param in WHERE clause somewhere
+    };
 
     /** the 5 second sleep function in Oracle SQL */
-    private static String SQL_ORACLE_TIME_SELECT =
-            "SELECT  UTL_INADDR.get_host_name('10.0.0.1') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.2') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.3') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.4') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.5') from dual";
+    private static String ORACLE_EXPENSIVE_SELECT =
+            "SELECT UTL_INADDR.get_host_name('10.0.0.1') from dual union SELECT UTL_INADDR.get_host_name('10.0.0.2') from dual union SELECT UTL_INADDR.get_host_name('10.0.0.3') from dual union SELECT UTL_INADDR.get_host_name('10.0.0.4') from dual union SELECT UTL_INADDR.get_host_name('10.0.0.5') from dual";
 
-    /** Oracle specific time based injection strings. each for 5 seconds */
-    // Note: <<<<ORIGINALVALUE>>>> is replaced with the original parameter value at runtime in these
-    // examples below (see * comment)
+    /** Oracle specific expensive injection strings. each for 5 seconds */
     // TODO: maybe add support for ')' after the original value, before the sleeps
-    private static String[] SQL_ORACLE_TIME_REPLACEMENTS = {
-        "(" + SQL_ORACLE_TIME_SELECT + ")",
-        "<<<<ORIGINALVALUE>>>> / (" + SQL_ORACLE_TIME_SELECT + ") ",
-        "<<<<ORIGINALVALUE>>>>' / (" + SQL_ORACLE_TIME_SELECT + ") / '",
-        "<<<<ORIGINALVALUE>>>>\" / (" + SQL_ORACLE_TIME_SELECT + ") / \"",
-        "<<<<ORIGINALVALUE>>>> and exists ("
-                + SQL_ORACLE_TIME_SELECT
+    private static String[] ORACLE_EXPENSIVE_PAYLOADS = {
+        "(" + ORACLE_EXPENSIVE_SELECT + ")",
+        ORIG_VALUE_TOKEN + " / (" + ORACLE_EXPENSIVE_SELECT + ") ",
+        ORIG_VALUE_TOKEN + "' / (" + ORACLE_EXPENSIVE_SELECT + ") / '",
+        ORIG_VALUE_TOKEN + "\" / (" + ORACLE_EXPENSIVE_SELECT + ") / \"",
+        ORIG_VALUE_TOKEN
```

Comment:
I edited all of these to use the constant instead of a literal, but that isn't strictly necessary.
Happy to roll back if people dislike it.

I also basically duplicated the payload arrays, which maybe isn't the best. Perhaps it should be a single array and the actually query insertion is handled with a replace as well?